lib/valkyrie/persistence.rb

```diff
@@ -31,7 +31,7 @@ module Persistence
     require 'valkyrie/persistence/composite_persister'
     require 'valkyrie/persistence/delete_tracking_buffer'
     require 'valkyrie/persistence/buffered_persister'
-    autoload :Postgres, 'valkyrie/persistence/postgres'
```

Comment:
Why remove autoload? This would mean users of the postgres adapter would have to put `require 'valkyrie/persistence/postgres'` in their...I dunno, `application.rb`?

Reply:
@tpendragon We are trying to remove the dependency on pg, with the autoload an exception gets raise on startup even if the user never utilizes pg

We added a line to the readme to show how to require

Reply:
That makes me feel like the autoload must be broken somehow. I'd really like to avoid forcing that weird explicit require. Maybe I can fiddle with it?

Reply:
When I spin up a new Rails application without pg, and this branch, but with the autoload, `rails c` works fine, as does `Valkyrie::Resource`, and `Valkyrie::Persistence::Solr::MetadataAdapter`. As soon as I call `Valkyrie::Persistence::Postgres::MetadataAdapter` it throws the error, though.

`rails s` also works. Could something in your code you're testing this with be calling the Postgres adapter?

Reply:
Does rspec just try to load everything?

Reply:
It shouldn't.

Reply:
@tpendragon You are correct.  I removed the PG tests and database cleaner and everything else works with the autoload.  I put it back in and adjusted the readme.